test/eventHandlers/workerInitRequest.test.ts

```diff
@@ -84,4 +85,33 @@ describe('workerInitRequest', () => {
             },
         });
     });
+
+    it('correctly loads package.json file', async () => {
+        const appDir = 'appDir';
+        const expectedPackageJson = {
+            type: 'module',
+        };
+        mock({
+            [appDir]: {
+                'package.json': JSON.stringify(expectedPackageJson),
+            },
+        });
+        await channel.initAppDir(appDir);

```

Comment:
I prefer if this file does its tests with `stream.addTestMessage(initMessage);` instead of calling a method on `channel`. Feels more realistic

Reply:
I did that. I couldn't get it to work without also copying the response message from the first test into a call to `stream.assertCalledWith()` so I extracted that message into a `Msg` namespace, similar to the tests I saw for `functionEnvironmentReloadRequest`. I don't know if there's an easier way to write the tests without needing that message though.

Reply:
Yes that is all by design and I like your changes. `stream.assertCalledWith` is supposed to validate the exact messages sent back and forth and so it's a good thing it failed without the response message